backend/src/packages/users/user.model.ts

```diff
@@ -32,6 +33,17 @@ class UserModel extends AbstractModel {
         ),
       },
     },
+    projects: {
+      relation: Model.HasOneRelation,
+      modelClass: ProjectModel,
+      join: {
+        from: 'users.id',
+        to: getJoinRelationPath<UserDetailsModel>(
+          DatabaseTableName.PROJECTS,
+          'userId',
+        ),
+      },
+    },
```

Comment:
```suggestion
```
you do not need it i think

Reply:
btw it is `HasManyRelation` as one user may have many projects